client/my-sites/plan-features/style.scss

```diff
@@ -69,6 +69,13 @@ $plan-features-sidebar-width: 272px;
 	height: 50px;
 }
 
+.plan-features__table.has-2-cols {
+	display: block;
+	max-width: 720px;
+	margin-left: auto;
```

Comment:
I think this can just be `margin: 0 auto`?

Reply:
Instead of adding `margin-left` and `margin-right`.

Reply:
margin-top is set by another rule and we want to set just left and right to auto